Project.toml

```diff
@@ -10,9 +10,11 @@ JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 MbedTLS = "739be429-bea8-5141-9913-cc70e7f3736d"
 Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
 SodiumSeal = "2133526b-2bfb-4018-ac12-889fb3908a75"
+URIs = "5c2747f8-b7ea-4ff2-ba2e-563bfd36b1d4"
 
 [compat]
-HTTP = "0.8, 0.9"
+HTTP = "0.9"
+URIs = "1.1, 1.3"
```

Comment:
```suggestion
URIs = "1.1"
```

Reply:
This pins URIs to 1.1, even though 1.2 and 1.3 have been released. What is the advantage of this?

Reply:
Gotcha. Reread the docs, thanks!